src/hera/v1/cron_workflow_service.py

```diff
@@ -0,0 +1,134 @@
+"""Holds the cron workflow service that supports client cron workflow creations"""
+from typing import Tuple
+
+from argo.workflows.client import (
+    CronWorkflowServiceApi,
+    V1alpha1CreateCronWorkflowRequest,
+    V1alpha1CronWorkflow,
+    V1alpha1CronWorkflowResumeRequest,
+    V1alpha1CronWorkflowSuspendRequest,
+)
+
+from hera.v1.client import Client
+from hera.v1.config import Config
+
+
+class CronWorkflowService:
+    """Argo cron workflow service for performing actions against cron workflows - creations, deletions, etc.
+
+    Parameters
+    ----------
+    domain: str
+        The Argo deployment domain to create cron workflows in.
+    token: str
+        The token to use for authentication purposes. Note that this assumes the Argo deployment is fronted with a
+        deployment/service that can intercept a request and check the Bearer token.
+    namespace: str = 'default'
+        The K8S namespace the cron workflow service creates cron workflows in.
+        This defaults to the `default` namespace.
+    """
+
+    def __init__(self, domain: str, token: str, namespace: str = 'default'):
+        self._domain = domain
+        self._namespace = namespace
+        api_client = Client(Config(domain), token).api_client
+        self.service = CronWorkflowServiceApi(api_client=api_client)
+
+    def create(self, cron_workflow: V1alpha1CronWorkflow, namespace: str = 'default') -> V1alpha1CronWorkflow:
+        """Creates given cron workflow in the argo server.
+
+        Parameters
+        ----------
+        cron_workflow: V1alpha1CronWorkflow
+            The cron workflow to create.
+        namespace: str = 'default'
+            The K8S namespace of the Argo server to create the cron workflow in.
+
+        Raises
+        ------
+        argo.workflows.client.ApiException: Raised upon any HTTP-related errors
+        """
+        return self.service.create_cron_workflow(
```

Comment:
I just realized the Argo SDK uses `create` vs. `submit`. Thank you for pointing it out via this implementation. We're going to have a `submit` on `workflow` and a `create` on cron now... that's fine, we'll change it in V2 🙂